### PR TITLE
Admin Generator (Future): set hideTopBar and remove AutomaticTitle

### PR DESCRIPTION
**Pull request by Ben-Ho**
this information is managed by some component higher up

### CODE REVIEW

**Comment by nsams** [comment]
Please make it easier for us to review this change by adding a screenshot of what you are hiding here. And describe (using screenshots if possible) situations where you need to hide it.

And can't you find a page in demo where this could be used and makes sense?

**Comment by Ben-Ho** [comment]
> Please make it easier for us to review this change by adding a screenshot of what you are hiding here. And describe (using screenshots if possible) situations where you need to hide it.

the problem occurs if the grid is used for example in a router-tabs component. I was also thinking if a more dynamic solution would be appropriate, but for now I only use generated grids either in router-tabs or as a page and not in both situations at the same time. (because different columns are needed if it's in context of a person or listing all where a person-column is required) To support reusing of the same grid in both scenarios handling via component-props would probably be better.

Screenshot 2024-05-22 at 21 14 10

> And can't you find a page in demo where this could be used and makes sense?

I could add a product-variants grid to the product-form router-tabs?

File: packages/admin/cms-admin/src/generator/future/generator.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     copyPaste?: boolean;
     readOnly?: boolean;
     filterProp?: boolean;
-    toolbar?: boolean;
+    toolbar?: false | { hideTopBar?: boolean; automaticTitleItem?: boolean };
```
Comment (nsams) [inline packages/admin/cms-admin/src/generator/future/generator.ts:62]: maybe rename `automaticTitleItem` to just `title`? (as the usage of `ToolbarAutomaticTitleItem` is an not-that-important implementation detail)

**Comment by nsams** [comment]
> I could add a product-variants grid to the product-form router-tabs?

yes, please

**Comment by Ben-Ho** [comment]
is already done in #2051

**Pull request closed without merging**